js/HelloWorldSceneAR.js

```diff
@@ -110,15 +119,20 @@ var styles = StyleSheet.create({
   },
 });
 
-const mapStateToProps = state => {
-  return {currentGame: state.game};
-};
+const mapStateToProps = state => ({
+  currentGame: state.game,
+  lockCombo: state.code,
```

Comment:
So line 124 here is I think an issue; I believe lock Combo has to be equal to: state.game.lockCombo